### PR TITLE
Remove django-jsonfield

### PR DESCRIPTION
**Pull request by toopy**

### CODE REVIEW

**Comment by toopy** [comment]
@k4nar OPM ?

File: django_ticketoffice/decorators.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ def guest_login(request, invitation):
         setattr(request, 'cache', {'invitation': invitation})
     request.user = GuestUser(invitation=invitation, invitation_valid=True)
     try:
-        request.user.id = invitation.data['user']
+        request.user.id = (invitation.data or {})['user']
```
Comment (Natim) [inline django_ticketoffice/decorators.py]: This is a bit cryptic, why not also expect TypeError below?

Comment (toopy) [reply django_ticketoffice/decorators.py]: Let's do this!

File: django_ticketoffice/decorators.py [hunk 2]
```diff
@@ -31,7 +31,7 @@ def guest_login(request, invitation):
     request.user = GuestUser(invitation=invitation, invitation_valid=True)
     try:
         request.user.id = invitation.data['user']
-    except KeyError:
+    except KeyError, TypeError:
```
Comment (Natim) [inline django_ticketoffice/decorators.py]: `except (KeyError, TypeError):`

Comment (Natim) [reply django_ticketoffice/decorators.py]: If not you are using the previous syntax; `except KeyError, err`

**Comment by toopy** [comment]
^ LGTR ?

**Pull request merged**